src/tests/quantity.rs

```diff
@@ -556,3 +491,82 @@ mod float {
         }
     }
 }
+
+mod non_complex {
+    storage_types! {
+        // Everything BUT complex
+        types: PrimInt, Ratio, Float, BigInt, BigUint;
+
+        use crate::tests::*;
+
+        mod f { Q!(crate::tests, super::V); }
+        mod k { Q!(crate::tests, super::V, (kilometer, kilogram, kelvin)); }
```

Comment:
The tests are for using a system of units with a different set of base units (km, kg, K) instead of (m, kg, K). There may be a fix for this in #229. I think that PR is pretty close and I just need to push it over the line.

Reply:
#229 was unrelated. A feature gate was missing!

```diff
diff --git a/src/tests/quantity.rs b/src/tests/quantity.rs
index a0f1383..90bd9b7 100644
--- a/src/tests/quantity.rs
+++ b/src/tests/quantity.rs
@@ -502,6 +502,7 @@ mod non_complex {
         mod f { Q!(crate::tests, super::V); }
         mod k { Q!(crate::tests, super::V, (kilometer, kilogram, kelvin)); }
 
+        #[cfg(feature = "autoconvert")]
         quickcheck! {
             #[allow(trivial_casts)]
             fn partial_cmp(l: A<V>, r: A<V>) -> bool {
```

Reply:
`#[cfg(...)]` can actually be moved up on top of the mod statement.